lib/simulator-management.js

```diff
@@ -214,5 +215,15 @@ async function shutdownOtherSimulators (currentDevice) {
   }
 }
 
+async function getInstalledBundleIds (device, candidateBundleIds) {
```

Comment:
since this is an oneliner then we could simply move it to `getInstalledWDBundleId`